tests/testthat/test-expr_array.R

```diff
@@ -17,38 +17,38 @@ test_that("arr$sum", {
   )
 })
 
-# TODO: reenable if the upstream issue is fixed <https://github.com/pola-rs/polars/issues/15646>
-# test_that("arr$max and arr$min", {
-#   skip_if_not(polars_info()$features$nightly)
 
-#   df = pl$DataFrame(
-#     ints = list(1:2, c(1L, NA_integer_), c(NA_integer_, NA_integer_)),
-#     floats = list(c(1, 2), c(1, NA_real_), c(NA_real_, NA_real_)),
-#     schema = list(
-#       ints = pl$Array(pl$Int32, 2),
-#       floats = pl$Array(pl$Float32, 2)
-#     )
-#   )
-#   # max ---
-#   expect_identical(
-#     df$select(pl$col("ints")$arr$max())$to_list(),
-#     list(ints = c(2L, 1L, NA_integer_))
-#   )
-#   expect_identical(
-#     df$select(pl$col("floats")$arr$max())$to_list(),
-#     list(floats = c(2, 1, NA_real_))
-#   )
+test_that("arr$max and arr$min", {
+  skip_if_not(polars_info()$features$nightly)
```

Comment:
I think we had this flag because it used to panick but I don't think it's needed anymore?

Reply:
Good point.
I think this could have been a bug of rustc (https://github.com/pola-rs/polars/issues/14359#issuecomment-1936910731), but it is worth seeing if it works with the current latest release.

Reply:
It seems working.